File: app/src/main/java/com/nizar/specialgirl/ContohCardView.java
package com.nizar.specialgirl;

import android.app.Activity;
import android.content.Intent;
import android.graphics.Color;
import android.os.Build;
import android.os.Bundle;
import android.support.v7.widget.CardView;
import android.view.View;
import android.view.Window;
import android.view.WindowManager;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

/**
 * Created by Poncoe on 29/10/17.
 */

public class ContohCardView extends Activity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.contoh_cardview);

        CardView contoh1 = (CardView) findViewById(R.id.card_view1);
        CardView contoh2 = (CardView) findViewById(R.id.card_view2);

        contoh1.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                Intent i = new Intent(view.getContext(), TidakTersedia.class);
                startActivity(i);
            }
        });
        contoh2.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                Intent i = new Intent(view.getContext(), TidakTersedia.class);
                startActivity(i);
            }
        });
    }

    public void onBackPressed() {
        finish();
    }
}

File: app/src/main/java/com/nizar/specialgirl/kotlin/ContohCardView.kt
package com.nizar.specialgirl.kotlin

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.support.v7.widget.CardView
import android.view.View
import android.view.Window

import com.nizar.specialgirl.R

/**
 * Created by Poncoe on 29/10/17.
 */

class ContohCardView : Activity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.contoh_cardview)

        val contoh1 = findViewById<View>(R.id.card_view1) as CardView
        val contoh2 = findViewById<View>(R.id.card_view2) as CardView

        contoh1.setOnClickListener { view ->
            val i = Intent(view.context, TidakTersedia::class.java)
            startActivity(i)
        }
        contoh2.setOnClickListener { view ->
            val i = Intent(view.context, TidakTersedia::class.java)
            startActivity(i)
        }
    }

    override fun onBackPressed() {
        finish()
    }
}